src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: url(../images/star.svg) center no-repeat;
+    @include star($star-size);
+    margin-right: 4px;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  @for $i from 1 through 5 {
+    &--#{$i} &__star:nth-child(-n + #{$i} ) {
+      background-image: url(..\images\star-active.svg);
+    }
+  }
+}
```

Comment:
Use nesting 
More details in [video](https://mate.academy/learn/css-advanced/sass#/video/925)